analyzers/PapasSim.py

```diff
@@ -61,7 +63,13 @@ def simparticle(ptc, index):
             pid = ptc.pdgid()
             simptc = Particle(tp4, vertex, charge, pid, index)
             pdebugger.info(" ".join(("Made", simptc.__str__())))
-            simptc.gen_ptc = ptc
+            #simptc.gen_ptc = ptc
+            #record that sim particle derives from gen particle
+            nodeid=simptc.uniqueid()
```

Comment:
no need to declare this var, especially since you redeclare it

Reply:
Done